frontend/src/components/LoginFormComponent.jsx

```diff
@@ -28,12 +29,16 @@ class LoginFormComponent extends Component {
             urlPath = "/api/auth/login/",
             requestBody = {username: this.state.username,
                         password: this.state.password};
+                        
         this.props.sendLogin(urlPath, requireAuth, requestBody)
         let loginAttemptNum = this.state.loginAttempt
```

Comment:
change to const if  you aren't changing it

Reply:
Okie I got rid of it and just used this.state.loginAttempt instead